pkg/event/http.go

```diff
@@ -110,5 +110,5 @@ func errorHandler(ctx context.Context, err error, w http.ResponseWriter) {
 		w.WriteHeader(http.StatusInternalServerError)
 	}
 
-	json.NewEncoder(w).Encode(map[string]interface{}{"error": err.Error()})
+	w.Write([]byte(ComponentName + "." + err.Error()))
```

Comment:
How do we ensure we do not leak any sensitive information here ?

Reply:
Right... we might use common-service like in other components and explicitly return a common_http.Error instance in above return statements.

Reply:
After a discussion with @harture , there is no risk of leaking information in this case.